developer-docs-site/docs/standards/fungible-asset.md

```diff
@@ -0,0 +1,407 @@
+
+#### Merging Fungible Assets
+
+Two FAs of the same type can be merged into a single struct that represents the accumulated value of the two  
+independently by calling:
+
+```rust
+public fun merge(dst_fungible_asset: &mut FungibleAsset, src_fungible_asset: FungibleAsset)
+```
+
+After merging, `dst_fungible_asset` will have all the amounts.
+
+#### Extracting Fungible Asset
+
+A FA can have amount deducted to create another FA by calling:
+
+```rust
+public fun extract(fungible_asset:& mut FungibleAsset, amount: u64): FungibleAsset
+```
+
+:::tip
+This function may produce FA with 0 amount, which is not usable. It is supposed to be merged with other FA or destroyed
+through `destroy_zero()` in the module.
+:::
+
+#### Withdraw
+
+The owner of a `FungibleStore` object that is not frozen can extract FA with a specified amount, by calling:
+
+```rust
+public fun withdraw<T: key>(owner: &signer, store: Object<T>, amount: u64): FungibleAsset
+```
+
+:::tip
+This function will emit a `WithdrawEvent`.
+:::
+
+#### Deposit
+
+Any entity can deposit FA into a `FungibleStore` object that is not frozen, by calling:
+
+```rust
+public fun deposit<T: key>(store: Object<T>, fa: FungibleAsset)
+```
+
+:::tip
+This function will emit a `DepositEvent`.
+:::
+
+#### Transfer
+
+The owner of a `FungibleStore` can directly transfer FA from that store to another if neither is frozen by calling:
+
+```rust
+public entry fun transfer<T: key>(sender: &signer, from: Object<T>, to: Object<T>, amount: u64)
+```
+
+:::tip
+This will emit both `WithdrawEvent` and `DepositEvent` on the respective `Fungibletore`s.
+:::
+
+## Events
+
+- `DepositEvent`: Emitted when FAs are deposited into a store.
+- `WithdrawEvent`: Emitted when FAs are withdrawn from a store.
+- `FrozenEvent`: Emitted when the frozen status of a fungible store is updated.
+
+```rust
+struct DepositEvent has drop, store {
+    amount: u64,
+}
+```
+
+```rust
+struct WithdrawEvent has drop, store {
+    amount: u64,
+}
+```
+
+```rust
+struct FrozenEvent has drop, store {
+    frozen: bool,
+}
+```
+
+# Primary `FungibleStore`
+
+Each `FungibleStore` object has an owner. However, an owner may possess more than one store. When Alice sends FA to
+Bob, how does she determine the correct destination? Additionally, what happens if Bob doesn't have a store yet?
+
+To address these questions, the standard has been expanded to define primary and secondary stores.
+
+- Each account owns only one undeletable primary store, the address of which is derived in a deterministic
+  manner. from the account address and metadata object address. If primary store does not exist, it will be created if
+  FA is going to be deposited by calling functions defined in `primary_fungible_store.move`
+- Secondary stores do not have deterministic address and theoretically deletable. Users are able to create as many
+  secondary stores as they want using the provided functions but they have to take care of the indexing by themselves.
```

Comment:
"take care of the indexing by themselves." is a bit vague to me. Any better way to explain this?

Reply:
I changed. it.